nbconvert/filters/strings.py

```diff
@@ -44,10 +44,10 @@
 
 
 def wrap_text(text, width=100):
-    """ 
+    """
```

Comment:
All these whitespace fixes were auto-made by my editor, I can try to revert them if it's noise

Reply:
All good imo. I do the same when touching files.